packages/yuudachi/src/events/interactionCreate.ts

```diff
@@ -117,14 +117,19 @@ export default class implements Event {
 
 						await interaction.editReply({ content: error.message, components: [] });
 					} catch (err) {
-						const error = err as Error;
-						logger.error(error, error.message);
+						const subError = err as Error;
+						logger.error(subError, subError.message);
 
 						if (interaction.isAutocomplete()) {
 							return;
 						}
 
-						await interaction.editReply({ content: error.message, components: [] });
+						if (subError.message.includes('Unknown Message')) {
```

Comment:
Is there any chance we can avoid string matching the error message? This seems somewhat janky.

Reply:
My final plan is to improve this part, i just couldn't find a good way. Maybe if We make follow-up as default for the catch handler we could avoid this, I will try

Reply:
Tested an solution in: 2448515